pinot-core/src/main/java/org/apache/pinot/core/query/reduce/GroupByDataTableReducer.java

```diff
@@ -218,59 +253,47 @@ private IndexedTable getIndexedTable(GroupBy groupBy, List<AggregationInfo> aggr
    * There will be 1 aggregation result per aggregation. The group by keys will be the same across all aggregations
    * @param brokerResponseNative broker response
    * @param dataSchema data schema
-   * @param aggregationInfos aggregations info
-   * @param groupBy group by info
-   * @param orderBy order by info
    * @param dataTables Collection of data tables
    */
-  private void setPQLGroupByOrderByResults(BrokerResponseNative brokerResponseNative, DataSchema dataSchema,
-      List<AggregationInfo> aggregationInfos, GroupBy groupBy, List<SelectionSort> orderBy,
+  private void setSQLGroupByInAggregationResults(BrokerResponseNative brokerResponseNative, DataSchema dataSchema,
```

Comment:
A comment here would be helpful for context -- something like this is needed for users that are using SQL GROUP BY ORDER BY but for backward compatibility, the results are still sent in old format. I believe this is what this function is doing. Right?

Reply:
Added a comment block, before the code breaks into the 4 paths. Lmk if it makes it clearer.